deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -175,7 +176,18 @@ def registry_value_cud():
 
     def registry_key_cud():
         pass
+    
+    def registry_value_create():
+        pass
 
+    def registry_value_update():
+        pass
+    
+    def registry_value_delete():
+        pass
+    
+    def create_values_content():
+        pass
```

Comment:
The functions are created functions that only work in windows, they do not work for Linux, but they must be created or the tests won't run because of imports errors